src/api/api.d.ts

```diff
@@ -201,6 +207,8 @@ export interface Repository {
 	getBranch(name: string): Promise<Branch>;
 	getBranches(query: BranchQuery): Promise<Ref[]>;
 	setBranchUpstream(name: string, upstream: string): Promise<void>;
+	getRefs(query: RefQuery, cancellationToken?: CancellationToken): Promise<Ref[]>;
```

Comment:
Same here. I would copy the whole file from the git extension.

Reply:
This file contains the API that GHPRI expects extension that register with it as a git provider to implement. Since this includes extensions other than the built in git extension, just copying over the file from the git extension would be misleading. I've copied over a little more of the file, but it is safer to leave it as is.